src/applications/appeals/995/tests/components/EvidenceVaRecords.unit.spec.jsx

```diff
@@ -79,7 +72,7 @@ describe('<EvidenceVaRecords>', () => {
     });
 
     // close modal by clicking method-assigned hidden button
-    fireEvent.click($('#test-method', container), mouseClick);
```

Comment:
What prompted this change? Was the mouseClick just not needed?

Reply:
`fireEvent.click` already uses a `mouseClick`, so it's duplicated effort. If you need a custom event, then use `fireEvent(..., new CustomEvent(...))`

Reply:
Ok cool, I assumed it was something like that, but was just curious. Thanks for the explanation!

Reply:
Actually, [I think the second parameter for named events](https://testing-library.com/docs/guide-events/) is ignored anyway...